db/migrate/20221212192830_create_itt_providers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateIttProviders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :itt_providers do |t|
+      t.text :legal_name, unique: true
+      t.text :operating_name
+      t.date :removed
```

Comment:
It would fit more in with the rest of the schema for this to be removed_at

Reply:
ok i can change that 👍

Reply:
think i had it this way at first to be honest